benchmark/testsystems/bookkeepers.py

```diff
@@ -45,138 +45,114 @@ def accumulate_shadow_work(self, x_0, v_0, n_steps):
 class EquilibriumSimulator():
     """Simulates a system at equilibrium."""
 
-    def __init__(self, platform, topology, system, positions, temperature, ghmc_timestep,
+    def __init__(self, platform, topology, system, positions, temperature, xcghmc_timestep,
                  burn_in_length, n_samples, thinning_interval, name):
 
         self.platform = platform
         self.topology = topology
         self.system = system
         self.positions = positions
         self.temperature = temperature
-        self.ghmc_timestep = ghmc_timestep
+        self.xcghmc_timestep = xcghmc_timestep
         self.burn_in_length = burn_in_length
         self.n_samples = n_samples
         self.thinning_interval = thinning_interval
         self.name = name
         self.cached = False
-
-        # get constraint tolerance
-        ghmc = GHMCIntegrator(temperature=self.temperature, timestep=self.ghmc_timestep)
-        self.constraint_tolerance = ghmc.getConstraintTolerance()
-        del (ghmc)
-        gc.collect()
+        self.constraint_tolerance = 1e-8
 
     def load_or_simulate_x_samples(self):
         """If we've already collected and stored equilibrium samples, load those
         Otherwise, collect equilibrium samples"""
-        self._path_to_samples = self.get_path_to_samples()
+        self._path_to_samples, self._path_to_box_vectors = self.get_path_to_samples_and_box_vectors()
```

Comment:
I don't think you need to use really long names like `get_path_to_samples_and_box_vectors`. Let's just consider that a *sample* is really any information that dynamically evolves with the simulation.

Are separate filepaths really needed for positions and box vectors? Isn't it much safer to store these together in one place/file?